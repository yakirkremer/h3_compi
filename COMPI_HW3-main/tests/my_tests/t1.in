void main(){
    auto a = 5;
    auto c = 5b;
    auto d = true;
    auto e = false;
    a = 6;
    c = 6b;
    d = false;
    a = 5b;
    a = (int) 5b;
    a = (byte) 5b;
    c = (byte) 10;
    a = true;
}