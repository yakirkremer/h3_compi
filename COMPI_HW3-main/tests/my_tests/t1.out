line 13: type mismatch
